app/controllers/api/base_controller.rb

```diff
@@ -199,7 +199,7 @@ def authorize
 
       unless authenticate
         count_login_failure
-        render_error('unauthorized', :status => :unauthorized, :locals => { :user_login => @available_sso.try(:user) })
+        render_error('unauthorized', status: :unauthorized, locals: { user_login: @available_sso.try(:user), message: _(@available_sso&.log_failed_auth_message)})
```

Comment:
If you want this translatable, you need to mark the messages as such. Otherwise they're not extracted. See inline comments elsewhere.

Also, I don't really see the point of `log_failed_auth_message`. You can still end up with `_(nil)` if `@available_sso` is `nil`. I'd recommend:

```suggestion
        message = @available_sso&.failed_auth_message
        render_error('unauthorized', status: :unauthorized, locals: { user_login: @available_sso.try(:user), message: message ? _(message) : ''})
```